Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1347,26 +1350,27 @@ public extension Purchases {
      * purchases and subscriptions across devices. Pass `nil` or an empty string if you want ``Purchases``
      * to generate this for you.
      *
-     * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
-     * RevenueCat's backend. Default is `false`.
+     * - Parameter purchasesAreCompletedBy: Set this to ``.myApp`` if you have your own IAP implementation and want to use only
+     * RevenueCat's backend. Default is ``.revenueCat``.
```

Comment:
```suggestion
     * RevenueCat's backend. Default is `.revenueCat`.
```

Reply:
The double quotes should enable auto-linking to the enum.

Reply:
Ah, I see, there was a warning! Fixed this. Thanks!